### PR TITLE
Fix `module 'apps.schedules.tasks.notify_about_empty_shifts_in_schedule' has no attribute 'apply_async'` `AttributeError`

### PR DESCRIPTION
**Pull request by joeyorlando**
# Which issue(s) this PR fixes

We've been seeing this `AttributeError` quite frequently for quite some time ([logs](https://ops.grafana-ops.net/explore?schemaVersion=1&panes=%7B%22oPl%22:%7B%22datasource%22:%22000000193%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bcluster%3D~%5C%22prod-%28eu-west-0%7Cus-central-0%29%5C%22,%20namespace%3D%5C%22amixr-prod%5C%22%7D%20%7C%3D%20%60AttributeError%28%5C%22module%20%27apps.schedules.tasks.notify_about_empty_shifts_in_schedule%27%20has%20no%20attribute%20%27apply_async%27%5C%22%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22000000193%22%7D,%22editorMode%22:%22code%22%7D%5D,%22range%22:%7B%22from%22:%22now-7d%22,%22to%22:%22now%22%7D%7D%7D&orgId=1))

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/schedules/tasks/__init__.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
-from .drop_cached_ical import drop_cached_ical_for_custom_events_for_organization, drop_cached_ical_task  # noqa: F401
-from .notify_about_empty_shifts_in_schedule import (  # noqa: F401
```
Comment (joeyorlando) [inline engine/apps/schedules/tasks/__init__.py:2]: I'm _fairly_ certain this leads to an issue for [these imports](https://github.com/grafana/oncall/blob/dev/engine/apps/schedules/tasks/refresh_ical_files.py#L5):
```python3
from apps.schedules.tasks import notify_about_empty_shifts_in_schedule, notify_about_gaps_in_schedule
```

It appears there is a naming clash between _modules_ and _functions_ with the same name here. This ☝️ `import` statement is importing the _modules_ `notify_about_empty_shifts_in_schedule` and `notify_about_gaps_in_schedule` from the `engine/apps/schedules/tasks` directory _not_ the similarly named functions that're imported into `engine/apps/schedules/tasks/__init__.py`.

File: engine/apps/schedules/tasks/refresh_ical_files.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 from apps.alerts.tasks import notify_ical_schedule_shift
 from apps.schedules.ical_utils import is_icals_equal
-from apps.schedules.tasks import notify_about_empty_shifts_in_schedule, notify_about_gaps_in_schedule
```
Comment (joeyorlando) [inline engine/apps/schedules/tasks/refresh_ical_files.py:5]: There _was_ a naming clash between _modules_ and _functions_ with the same name here. This ☝️ `import` statement is importing the _modules_ `notify_about_empty_shifts_in_schedule` and `notify_about_gaps_in_schedule` from the `engine/apps/schedules/tasks` directory _not_ the similarly named functions that're imported into `engine/apps/schedules/tasks/__init__.py`.

File: engine/settings/celery_task_routes.py [hunk 3]
```diff
@@ -33,15 +33,15 @@
     "apps.schedules.tasks.notify_about_gaps_in_schedule.check_empty_shifts_in_schedule": {"queue": "default"},
     "apps.schedules.tasks.notify_about_gaps_in_schedule.start_notify_about_gaps_in_schedule": {"queue": "default"},
     "apps.schedules.tasks.notify_about_gaps_in_schedule.check_gaps_in_schedule": {"queue": "default"},
-    "apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_empty_shifts_in_schedule": {"queue": "default"},
+    "apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_gaps_in_schedule_task": {"queue": "default"},
```
Comment (joeyorlando) [inline engine/settings/celery_task_routes.py:36]: `apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_empty_shifts_in_schedule` is a typo, it does not correspond to a real function.

I've simply moved `apps.schedules.tasks.notify_about_gaps_in_schedule.notify_about_gaps_in_schedule` up in the dict to be alongside the other `apps.schedules.tasks.notify_about_gaps_in_schedule` module tasks

File: engine/apps/schedules/tasks/drop_cached_ical.py
Comment (joeyorlando) [inline engine/apps/schedules/tasks/drop_cached_ical.py:1]: without this change and with the fix in `engine/apps/schedules/tasks/__init__.py` this file was resulting in a circular import, hence moving the import inside of the function where it is being used

**Review by mderynck — APPROVED** [review]

**Pull request merged**